app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -17,7 +17,7 @@ def visibility_options
       end
 
       def update(attributes)
-        manage_grants = attributes[:access_grants_attributes].select { |x| x[:access] == 'manage' }
+        manage_grants = Array.wrap(attributes[:access_grants_attributes]).select { |x| x[:access] == 'manage' }
```

Comment:
Unrelated to your PR, @tdonohue, but I wonder if we ought to rename `x` to be more intention-revealing?